client/src/pages/Intro.jsx

```diff
@@ -1,21 +1,19 @@
-import { ThemeProvider } from "styled-components"
-import Glass from "../assets/Glass"
-import { Flex } from "../components/styles/Flex"
-
+import { ThemeProvider } from "styled-components";
+import Glass from "../assets/Glass";
+// import { Flex } from "../components/styles/Flex"
+import { PagesStyle } from "../style";
 const theme = {
-  colors: {
-
-  },
-  mobile: "798px"
-}
+  colors: {},
+  mobile: "798px",
+};
 
 const Intro = () => {
   return (
     <ThemeProvider theme={theme}>
-    <Flex>
```

Comment:
why did you remove this?

Reply:
עשיתי קומפוננטה חדשה- קונבנציה לכל הדפים הלבנים שמתלבשים על בדף הלבן. הPagesStyle מחליף את הFlex ואפשר להוסיף את הסוג דיב הזה לכל מי שעבד על הדפים הלבנים